locale/en/blog/weekly-updates/weekly-update.2016-11-04.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Weekly Update - November 4th, 2016
+author: Minwoo Jung (github.com/JungMinu)
+date: 2016-11-04T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-11-04
+layout: blog-post.hbs
+---
+
+### Node v7.1.0 proposal
+
+* [Node v7.1.0 proposal](https://github.com/nodejs/node/pull/9438). The complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```

Comment:
Nit: I would leave out `from previous releases` and just write `The complete changelog can be found...`